src/utilities/theme/utils.ts

```diff
@@ -225,6 +225,27 @@ function overrides() {
     [toCssCustomPropertySyntax('focusRingContent')]: "''",
     [toCssCustomPropertySyntax('borderRadiusBase')]: rem('4px'),
     [toCssCustomPropertySyntax('borderRadiusWide')]: rem('8px'),
+    [toCssCustomPropertySyntax('borderBox')]: 'border-box',
+    [toCssCustomPropertySyntax('defaultBannerBorder')]: buildBannerBorder(
```

Comment:
Early days, but other items have the component name first (`buttonFontWeight`, `badgeMixBlendMode`). I think we should follow the same system with these so things are grouped when doing alphabetic listing which makes them a bit easier to skim through. So `bannerSuccessBorder`, `bannerHighlightBorder` etc